code/controllers/master.dm

```diff
@@ -60,14 +60,17 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 
 /datum/controller/master/New()
 	// Highlander-style: there can only be one! Kill off the old and replace it with the new.
-	subsystems = list()
-	if (Master != src)
```

Comment:
1: unrelated change. =P

2: `New()` is not `new()`. There is no reason to assume that `New()` will only be called by `new()`. Between children double calling `..()`, byond bugs, and admin proc calls, `New()` can get called well after a thing is `new()`ed.  (`New()` is proc `0` in a objects proc list, so a null reference error in byond can trigger it, there was once a bug where invalid procs in call()() triggered calls to `New()` (and invalid src would trigger a call to `world`/`New()`))